test/mlir/onnx/onnx_shape_inference.mlir

```diff
@@ -1238,7 +1238,7 @@ func @test_shape(%arg0: tensor<?x3x2xf32>) -> tensor<*xi64> {
   return %0 : tensor<*xi64>
 
   // CHECK-LABEL: test_shape
-  // CHECK: [[RES:%.+]] = "onnx.Shape"(%arg0) : (tensor<?x3x2xf32>) -> tensor<3xi64>
+  // CHECK: [[RES:%.+]] = "onnx.Constant"() {value = dense<[-1, 3, 2]> : tensor<3xi64>} : () -> tensor<3xi64>
```

Comment:
Why did this test and the one below it need to be changed?

Reply:
My changes changes the behavior of two existing tests. In both cases the Shape Op Node is replaced with a Constant Op because of the folding.

Reply:
Ah right, makes sense